python/nav/web/machinetracker/utils.py

```diff
@@ -197,17 +197,8 @@ def __common(self):
         if not self.input.get('days', False):
             self.input['days'] = 7
 
-    def __prefix(self):
-        try:
-            ip = Prefix.objects.get(id=self.input['prefixid'])
-        except Prefix.DoesNotExist:
-            return None
-        self.input['ip_range'] = ip.net_address
```

Comment:
Yuck

Reply:
Ideally, I'd want to get rid of this entire class, but no time for that now.